bugbot/rules/lot_of_comments.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SeveralComments(BzCleaner):
+    def __init__(self):
+        super(SeveralComments, self).__init__()
+        self.nweeks = utils.get_config(self.name(), "weeks_lookup")
+        self.comments = utils.get_config(self.name(), "number_comments")
```

Comment:
We recently started using the constructor parameters to configure the rules instead of the config file (see https://github.com/mozilla/bugbot/issues/1437). With that, we have the configs close to the rule, and we can easily document the configurations. Here is an example:

https://github.com/mozilla/bugbot/blob/a929257d756715933ec2672ac63da38ed66eeeaf/bugbot/rules/patch_closed_bug.py#L17-L26


Configurations related to running the rule, escalation, and receivers should still live in the config file (e.g., `additional_receivers`, and `must_run`).